cogs/qrchecker/constants.py

```diff
@@ -1,3 +1,5 @@
 KEY_ENABLED = "enabled"
+KEY_MAX_IMAGE_PIXELS: str = "maxImagePixels"
```

Comment:
Can strip the type hinting for this constant.

```suggestion
KEY_MAX_IMAGE_PIXELS = "maxImagePixels"
```